scripts/deploy_appstore.sh

```
@@ -0,0 +1,311 @@
+print_info "Loading iOS shared configuration..."
+source secrets/shared_keys.env
+
+# Validate App Store Connect API key configuration
+if [ -z "$APPSTORE_KEY_ID" ] || [ -z "$APPSTORE_ISSUER_ID" ]; then
+    print_error "App Store Connect API credentials not configured"
+    print_info "Update secrets/shared_keys.env with APPSTORE_KEY_ID and APPSTORE_ISSUER_ID"
+    exit 1
+fi
+
+# Load Match password
+export MATCH_PASSWORD=$(cat secrets/.match_password)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)